manim/utils/config_utils.py

```diff
@@ -521,10 +508,72 @@ def finalized_configs_dict():
     return {section: dict(config[section]) for section in config.sections()}
 
 
-def _subcommands_exist(ignore=[]):
+def _subcommand_name(ignore=()):
+    """Goes through sys.argv to check if any subcommand has been passed,
+    and returns the first such subcommand's name, if found.
+
+    Parameters
+    ----------
+    ignore : Iterable[:class:`str`], optional
+        List of NON_ANIM_UTILS to ignore when searching for subcommands, by default []
+
+    Returns
+    -------
+    Union[:class:`str`, :class:`None`]
```

Comment:
1. None is not a class, so it would be just None;
2. Union[..., None] should always be written as Optional[...]. 
```suggestion
    Optional[:class:`str`]
```